tests/integrations/realcluster/real_cluster.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2024 TiKV Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package realcluster
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/pingcap/log"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/zap"
+)
+
+type realClusterSuite struct {
+	suite.Suite
+
+	clusterCnt int
+	suiteName  string
+}
+
+var tiupBin = os.Getenv("HOME") + "/.tiup/bin/tiup"
+
+// SetupSuite will run before the tests in the suite are run.
+func (s *realClusterSuite) SetupSuite() {
+	t := s.T()
+
+	dataDir := filepath.Join(os.Getenv("HOME"), ".tiup", "data", "pd_real_cluster_test_"+s.suiteName+"_*")
```

Comment:
Should we use a tmp dir?

Reply:
This directory is the default dir of tiup.

Reply:
https://docs.pingcap.com/tidb/stable/tiup-playground#specify-a-tag-when-starting-the-tidb-cluster